src/pages/global-styles.js

```diff
@@ -108,7 +108,14 @@ injectGlobal`
   }
 
   a:hover {
-    color: #797d80;
+    color: #ffffff;
```

Comment:
Not all anchor tags are buttons, so it doesn't make sense to use a global style for background color on all anchor tags. This should really be a part of a Button component. One of which exists, as well as the StyledLink that can be styled like a button.